### ISSUE TITLE
Non-drug treatments (dietary therapies, behavior programs, physical therapy, etc.) aren't being searched with "treats" queries

### ISSUE THREAD

**Issue by karafecho** [NCATSTranslator/Feedback#32]
Translator often doesn’t explore queries outside of certain Biolink categories / entity types such as biolink:Disease, biolink:Gene, biolink:ChemicalEntity. Yet, for the October 2022 QotM, adoption of a ketogenic diet was considered an acceptable treatment option. How would Translator identify non-drug treatments such as dietary therapies? biolink:ClinicalIntervention? biolink:Treatment? Are there Translator knowledge sources to support such queries?

Note that this is not a new issue. In fact, multiple SMEs have asked for non-drug therapies in QotM calls, relay meeting sessions, etc.

**Comment by GitHubbit** [comment]
The Multiomics Clinical Trials KP could potentially answer. Currently, the core triple supported is [Condition <---related_to---Intervention]. So Conditions are Biolink:Disease, and Interventions are Biolink:Treatment.
Types of interventions in this KP are definitely not limited to medicines. There are behavioral programs, essential oils, dietary changes, etc.

**Comment by karafecho** [comment]
ICEES KG also can respond to certain queries involving biolink:ClinicalIntervention, e.g., biolink:Disease - RWE - biolink:ClinicalIntervention.

**Comment by sierra-moxon** [comment]
SEMMED and clinical groups have this data.
creative mode focused on chemical treatments not non-chemical treatments.

**Comment by sandrine-m** [comment]
Hi @karafecho , thank you for filling the creative mode registry with this additional proposal for extension of the MVP1 question. The UCWG created a meta-template scoring system based on requirements by NCATS. You can see it [here](https://docs.google.com/spreadsheets/d/1gft4SdKRe6aLkRy9prZIP1lHyYvrLUmSKzQRouo7F2k/edit?usp=sharing). Unfortunately the query is not ranked the highest as this query does not use qualifiers (requirement).

**Comment by karafecho** [comment]
I updated your comment, just to correct the record, so yes, we can close the issue. Thanks!

**Comment by karafecho** [comment]
Reopening as discussed ...

**Comment by andrewsu** [comment]
When I execute this query asking for diseases treated by exercise, I get many results as expected, including MS, stroke, spinal cord injury, etc. ([ARAX link](https://arax.ncats.io/?r=8d85bbb4-2085-4ad8-a71a-b4dd5099c4a0)):

```
{
  "message": {
    "query_graph": {
      "nodes": {
        "n0": {
           "ids": ["UMLS:C0015259"],
           "categories": ["biolink:Procedure"]
        },
        "n1": {
          "categories": [
            "biolink:DiseaseOrPhenotypicFeature"
          ]
        }
      },
      "edges": {
        "e01": {
          "object": "n1",
          "subject": "n0",
          "predicates": ["biolink:treats"]
        }
      }
    }
  }
}
```

Note that it's essential right now that `n0` includes `"categories": ["biolink:Procedure"]` because prod node normalizer does not currently resolve `UMLS:C0015259` ([link](http://nodenorm.transltr.io/1.3/get_normalized_nodes?curie=UMLS:C0015259)).  That is likely a temporary issue since dev node normalizer does resolve as expected ([link](https://nodenormalization-sri.renci.org/1.3/get_normalized_nodes?curie=UMLS:C0015259)).

**Comment by karafecho** [comment]
Wow! This is great! Thanks for the insights, Andrew.

**Comment by gaurav** [comment]
> That is likely a temporary issue since dev node normalizer does resolve as expected ([link](https://nodenormalization-sri.renci.org/1.3/get_normalized_nodes?curie=UMLS:C0015259)).

Correct! We're holding back a Prod NodeNorm update because of the Relay; I'll put in the request tomorrow and hopefully Prod NodeNorm will return the same results as Dev NodeNorm sometime next week. (Note that [Test NodeNorm](https://nodenorm.test.transltr.io/1.3/get_normalized_nodes?curie=UMLS:C0015259) resolves `UMLS:C0015259` correctly).

**Comment by andrewsu** [comment]
... just for completeness, I also ran the query for Ketogenic diet (UMLS:C0259972) and the expected results are returned: https://arax.ncats.io/?r=968584ce-150f-40f8-be2c-4dfe459b762c

Unless anyone has objections, I think this issue can be closed...

**Comment by karafecho** [comment]
Thanks for running the ketogenic diet query, @andrewsu. You may want to follow up with Drs. Friedman and Forero, as the results look kind of interesting, at least from a cursory review. The edges I looked at were all 'treats' edges from SemMedDB, but the publication support seemed reasonable.

**Comment by sandrine-m** [comment]
@sierra-moxon is there another issue here that I missed or shall we close this ticket? Thanks
